### DISCUSSION TITLE
PB14 and PC9 of STM32G070RBT6 can't use PWM at the same time

### DISCUSSION THREAD

**Discussion by sunnyguhz** [stm32duino/Arduino_Core_STM32#2225] [Q&A] [answered]
I set PWM for PB14 and PC9 of STM32G070RBT6, but only one of them can work. 
What can I do to make them work simultaneously?

```
void setup() {
  analogWriteResolution(12);  
  analogWriteFrequency(10000);  
  pinMode(PB14, OUTPUT);
  pinMode(PC9, OUTPUT);
}

void loop() {
  analogWrite(PB14, 2048);
  analogWrite(PC9, 1024);
  delay(1);
}
```

**Comment by fpistm** [comment, marked as answer]
Hi @sunnyguhz 
This is normal:
https://github.com/stm32duino/Arduino_Core_STM32/blob/63d30964d41809b3f704f0cab8c5b64870fdcc0d/variants/STM32G0xx/G070RBT/PeripheralPins.c#L123C4-L124
https://github.com/stm32duino/Arduino_Core_STM32/blob/63d30964d41809b3f704f0cab8c5b64870fdcc0d/variants/STM32G0xx/G070RBT/PeripheralPins.c#L134C5-L135

Both pin used TIM1 by default.
Simply suffix one by `_ALT1` to not use the same TIM instance

```
  analogWrite(PB14, 2048);
  analogWrite(PC9_ALT1, 1024);
```
